nss-c31-unusual-prestige/nutshell
# News fetch display

# Description

Created function to pull News Articles from database and display each to DOM.

Applies to issue # 4

## Type of change





# Testing Instructions

Clone down branch. Make sure to add news dummy data into your database. Make sure newsDomManager is imported to main.js and call listAllNews function. Run grunt and articles should appear on DOM.



# Checklist:
